### PR TITLE
Brojd/connect integration to snow

### PR DESCRIPTION
**Pull request by brojd**
# What this PR does

https://github.com/grafana/oncall/assets/12073649/0dad62c2-d722-4f5b-aee6-549dc97902cd

## Checklist

- [x] Unit, integration, and e2e (if applicable) tests updated
- [x] Documentation added (or `pr:no public docs` PR label added if not required)
- [x] `CHANGELOG.md` updated (or `pr:no changelog` PR label added if not required)

### CODE REVIEW

File: grafana-plugin/src/pages/integration/OutgoingTab/ConnectIntegrationModal.tsx [hunk 1]
```diff
@@ -1,29 +1,103 @@
...
+
+  const onChange = (integration: ApiSchemas['AlertReceiveChannel'], checked) => {
+    if (checked) {
+      setSelectedIntegrations((integrations) => [...integrations, integration]);
+    } else {
+      setSelectedIntegrations((integrations) => integrations.filter(({ id }) => id !== integration.id));
+    }
+  };
+
+  const onConnect = () => {};
```
Comment (brojd) [inline grafana-plugin/src/pages/integration/OutgoingTab/ConnectIntegrationModal.tsx:51]: waiting for backend

File: grafana-plugin/src/pages/integration/OutgoingTab/ConnectedIntegrationsTable.tsx [hunk 2]
```diff
@@ -1,57 +1,81 @@
...
-  return (
-    <GTable
-      emptyText={FAKE_INTEGRATIONS ? 'No integrations found' : 'Loading...'}
-      rowKey="id"
-      columns={getColumns(props)}
-      data={FAKE_INTEGRATIONS}
-    />
-  );
-};
+    const columns = useMemo(
```
Comment (teodosii) [inline grafana-plugin/src/pages/integration/OutgoingTab/ConnectedIntegrationsTable.tsx:27]: Why the use of `usememo`?

File: grafana-plugin/src/pages/integration/OutgoingTab/OtherIntegrations.tsx [hunk 3]
```diff
@@ -24,7 +24,7 @@ export const OtherIntegrations = observer(() => {
             </Button>
           </HorizontalGroup>
         }
-        content={<ConnectedIntegrationsTable allowDelete />}
+        content={<ConnectedIntegrationsTable allowDelete allowBacksync tableProps={{ data: [] }} />}
```
Comment (brojd) [inline grafana-plugin/src/pages/integration/OutgoingTab/OtherIntegrations.tsx:27]: waiting for backend

File: grafana-plugin/src/pages/integration/OutgoingTab/ConnectedIntegrationsTable.tsx (hunk 2 above)
Comment (brojd) [reply grafana-plugin/src/pages/integration/OutgoingTab/ConnectedIntegrationsTable.tsx:27]: yeah we don't need yet, will remove 👍

Comment (brojd) [reply grafana-plugin/src/pages/integration/OutgoingTab/ConnectedIntegrationsTable.tsx]: done

**Review by teodosii — APPROVED** [review]

**Pull request merged**